fontbe/src/features.rs

```diff
@@ -113,39 +125,82 @@ impl<'a> FeaVariationInfo<'a> {
     }
 }
 
+/// When generating mark features it's handy to know what is a base and what is a mark
+#[derive(Copy, Clone, Debug, PartialEq)]
+enum AnchorType {
+    Base,
+    Mark,
+}
+
+impl AnchorType {
+    fn group_name(&self, glyph_name: GlyphName) -> GlyphName {
+        match self {
+            AnchorType::Base => glyph_name,
+            AnchorType::Mark => glyph_name.as_str()[1..].into(),
```

Comment:
so looking below, it seems that every time we have an `AnchorType` we call this method several times. Would it make sense to compute that once when we create the enum, and store it there?

Reply:
Changed to not take ownership